docs/api.md

```diff
@@ -33,8 +33,8 @@ Reading anndata’s native formats `.h5ad` and `zarr`.
 .. autosummary::
    :toctree: generated/
 
-   read_h5ad
-   read_zarr
+   io.read_h5ad
+   io.read_zarr
```

Comment:
maybe we should document that those are also available as `anndata.read_*`?

Reply:
I think we shouldn't.   I think we should be encouraging people to use `io`.  We won't break their code for a long time if they continue to use `anndata.read_{zarr,h5ad}` (currently these are the only two from the top level _not_ deprecated).  But I still think we should be encouraging this.  Maybe a note in their docstrings then?

Reply:
I guess since we import it by default, it’s just 3 more characters to type and no other change.